backend/db/mongo.go

```diff
@@ -20,13 +20,19 @@ func (c *Database) insertUser(user User) (string, error) {
 	return "", errors.New("Invalid id created")
 }
 
-// insert NewsPiece into mongo
-// func (c *Database) insertArticle(article Article) error {
-// collection := c.database.Collection("articles")
-// to set the id ourselves we need to
-// insertResult, err := collection.InsertOne(context.TODO(), &Article{})
-// 	return errors.New("Invalid id created")
-// }
+// insert NewsPiece associated with User into mongo
+func (c *Database) insertArticle(article Article) (string, error) {
+	collection := c.database.Collection("articles")
```

Comment:
We can keep them hardcoded for now, we might be able to do some work with env variables, but I don't mind the hardcoding here - we can add a followup later that makes them constants